cogboard-webapp/src/components/WidgetForm/index.js

```diff
@@ -29,8 +39,14 @@ const WidgetForm = ({ renderActions, ...initialFormValues }) => {
     setTabValue(newValue);
   };
 
+  useEffect(() => {
+    const validationSchema = createWidgetValidationSchema(values.type, generalFields, constraints)
+    setValidationSchema(validationSchema)
+    // eslint-disable-next-line
+  }, [values.type])
```

Comment:
Add all dependencies, react will handle which will be updated - values.type in this case anyway, because rest are the same :)